azuredevops/internal/service/audit/resource_auditstream_azuremonitorlogs.go

```diff
@@ -0,0 +1,59 @@
+package audit
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/microsoft/azure-devops-go-api/azuredevops/v6/audit"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/utils/converter"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/utils/tfhelper"
+)
+
+// ResourceAuditStreamAzureEventGrid schema and implementation for Azure EventHub audit resource
+func ResourceAuditStreamAzureMonitorLogs() *schema.Resource {
+	r := genBaseAuditStreamResource(flattenAuditStreamAzureMonitorLogs, expandAuditStreamAzureMonitorLogs)
+
+	r.Schema["workspace_id"] = &schema.Schema{
+		Type:         schema.TypeString,
+		Required:     true,
+		DefaultFunc:  schema.EnvDefaultFunc("AZDO_AUDIT_AZURE_MONITORLOGS_WORKSPACE_ID", nil),
+		ValidateFunc: validation.StringIsNotWhiteSpace,
```

Comment:
`workspace_id` is a UUID
```suggestion
		ValidateFunc: validation.IsUUID,
```